### PR TITLE
Species search cell type wheel not clickable

### PR DESCRIPTION
**Pull request by lingyun1010**
In `scxa-cell-type-wheel` component, we hard-coded to make a gene heatmap search at node level 4, that's right for cell type metadata search, rather than for `species` metadata search. We need to make a condition to check if the search term is `species` or not by fetching all possible species from the `scxa-cell-type-wheel-experiment-heatmap` component.

If you want to test it locally, please remember to build a local `scxa-cell-type-wheel` package by typing `npm pack` under its directory and install with the absolute/relative directory in `scxa-cell-type-wheel-experiment-heatmap` component.

I also created a html demo page for `Mus muscuslus` search term in `scxa-cell-type-wheel` component, with altered json response, by removing the incorrect entry with `Homo sapiens`. This entry issue will be solved by data side as far as I know.

### CODE REVIEW

**Review by upendrakumbham — APPROVED** [review]
LGTM! I tested in my local, 3rd level(cell types) in the cell type wheel is clickable.

File: packages/scxa-cell-type-wheel-experiment-heatmap/src/CellTypeWheelExperimentHeatmap.js [hunk 1]
```diff
@@ -14,17 +14,45 @@ const MarkerGeneHeatmapFetchLoader = withFetchLoader(MarkerGeneHeatmap)
...
+  const [error, setError] = useState(null)
+
+  useEffect(() => {
+    // Define the async function inside useEffect
+    const fetchData = async () => {
+      const url = URI(props.searchFormResource, props.host).toString()
+      try {
+        const response = await fetch(url)
+        if (!response.ok) {
+          throw new Error('Network response was not ok')
```
Comment (ke4) [inline packages/scxa-cell-type-wheel-experiment-heatmap/src/CellTypeWheelExperimentHeatmap.js:28]:
```suggestion
          throw new Error('Network response was not ok. Please try again later.')
```

**Review by ke4 — CHANGES_REQUESTED** [review]
I added a small suggestion, but otherwise it looks fine with me.

**Review by ke4 — APPROVED** [review]
LGTM

**Pull request merged**